pytket/binders/passes.cpp

```diff
@@ -343,7 +343,9 @@ PYBIND11_MODULE(passes, m) {
       "FullPeepholeOptimise", &FullPeepholeOptimise,
       "Performs peephole optimisation including resynthesis of 2- and 3-qubit "
       "gate sequences, and converts to a circuit containing only CX and TK1 "
-      "gates.");
+      "gates."
+      "\n\n:param: allow_swaps whether to allow implicit wire swaps",
```

Comment:
```suggestion
      "\n\n:param: allow_swaps whether to allow implicit wire swaps.",
```

Reply:
Done (and fixed a few other instances too).